libs/mark-flow-ui/src/components/voter_screen.tsx

```diff
@@ -8,20 +8,28 @@ import {
   LanguageSettingsScreen,
   Main,
   Screen,
+  VoterHelpButton,
   VoterSettings,
   useScreenInfo,
 } from '@votingworks/ui';
 import { SizeMode } from '@votingworks/types';
 
+import { assert } from '@votingworks/basics';
 import { VoterSettingsButton } from './voter_settings_button';
 
+export interface VoterHelpScreenProps {
+  onClose: () => void;
+}
+export type VoterHelpScreenType = React.ComponentType<VoterHelpScreenProps>;
+
 export interface VoterScreenProps {
   actionButtons?: React.ReactNode;
   breadcrumbs?: React.ReactNode;
   children: React.ReactNode;
   centerContent?: boolean;
   padded?: boolean;
   hideMenuButtons?: boolean;
+  VoterHelpScreen?: VoterHelpScreenType;
```

Comment:
I'm passing in the help screen contents dynamically to libs/mark-flow-ui because VxMark will need different help screen contents then VxMarkScan when the time comes